src/Processors/Formats/Impl/NativeORCBlockInputFormat.cpp

```diff
@@ -1312,7 +1312,7 @@ static ColumnWithTypeAndName readColumnWithEncodedStringOrFixedStringData(
     size_t rows = orc_str_column.numElements;
     const auto & orc_dict = *orc_str_column.dictionary;
     if (orc_dict.dictionaryOffset.size() <= 1)
-        return {internal_type->createColumn(), internal_type, column_name};
+        return {internal_type->createColumnConstWithDefaultValue(rows), internal_type, column_name};
```

Comment:
```suggestion
    {
        auto result_column = internal_type->createColumn();
        result_column->insertManyDefaults(rows);
        return {result_column, internal_type, column_name};
    }
```

Using constant columns in such place is dangerous. For example, if we have some complex data type like Tuple/Array/Map we cannot have constant nested columns.